From ab44384df40dbb99df4f0759d1af45dc162b18b2 Mon Sep 17 00:00:00 2001
From: Jason King <jason.king@joyent.com>
Date: Fri, 14 Dec 2018 18:22:33 -0600
Subject: [PATCH] OS-7455 Force physical block size for Samsung 860 SSD
 Reviewed by: Dan McDonald <danmcd@joyent.com> Approved by: Robert Mustacchi
 <rm@joyent.com>

---
 overlay/generic/kernel/drv/sd.conf | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/overlay/generic/kernel/drv/sd.conf b/overlay/generic/kernel/drv/sd.conf
index fce0d72c..b0aebdb5 100644
--- a/overlay/generic/kernel/drv/sd.conf
+++ b/overlay/generic/kernel/drv/sd.conf
@@ -79,4 +79,5 @@ sd-config-list=
 	"DELL    PERC/6i", "cache-nonvolatile:true",
 	"ATA     Samsung SSD 830", "physical-block-size:4096",
 	"ATA     Samsung SSD 840", "physical-block-size:4096",
-	"ATA     Samsung SSD 850", "physical-block-size:4096";
+	"ATA     Samsung SSD 850", "physical-block-size:4096",
+	"ATA     Samsung SSD 860", "physical-block-size:4096";
-- 
2.21.0

